101-servicebus-create-namespace/azuredeploy.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "serviceBusNamespaceName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the Service Bus namespace"
+            }
+        },
+        "serviceBusSku": {
+            "type": "string",
+            "allowedValues": [
+                "Basic",
+                "Standard"
+            ],
+            "defaultValue": "Standard",
+            "metadata": {
+                "description": "The messaging tier for service Bus namespace"
+            }
+        },
+        "serviceBusApiVersion": {
```

Comment:
The parameter file had a default api-version. Besides our service returns back the correct API versions in case invoked with the wrong one. Beside when people use the AzureDeploy  button, they could just edit the AOI version if they wanted to rather than modifying the template itself.